container/libcontainer/handler.go

```diff
@@ -728,8 +742,13 @@ func setMemoryStats(s *cgroups.Stats, ret *info.ContainerStats) {
 		ret.Memory.HierarchicalData.Pgmajfault = v
 	}
 
+	inactiveFileKeyName := "total_inactive_file"
+	if cgroups.IsCgroup2UnifiedMode() {
+		inactiveFileKeyName = "inactive_file"
```

Comment:
is this actually the same as total_inactive_file?  I thought total included all children cgroup's inactive_file?

Reply:
in cgroupv2 there is only `inactive_file` that has the same semantic as `total_inactive_file` in v1.  

I've verified it locally with:

```
#  mkdir -p /sys/fs/cgroup/a/b/c/d
...(move process to the new cgroup, enable the memory controller and open/read some random files)...
/ # grep inactive_file /sys/fs/cgroup/a/b/c/d/memory.stat
inactive_file 405504
/ # grep inactive_file /sys/fs/cgroup/a/b/c/memory.stat
inactive_file 405504
/ # grep inactive_file /sys/fs/cgroup/a/b/memory.stat
inactive_file 405504
/ # grep inactive_file /sys/fs/cgroup/a/memory.stat
inactive_file 405504
```